lib/membrane_audio_mixer.ex

```diff
@@ -320,11 +321,10 @@ defmodule Membrane.AudioMixer do
     )
   end
 
-  defp mix_or_redemand(size, time_frame, context, state) do
+  defp mix_and_redemand(size, time_frame, context, state) do
     if size >= time_frame do
       {actions, state} = mix_and_get_actions(context, state)
-      actions = if actions == [], do: [redemand: :output], else: actions
-      {actions, state}
+      {actions ++ [redemand: :output], state}
```

Comment:
We can leave it like that or redemand only when mixed audio's byte size is smaller than the input audio's size.